src/readers/morphologyHDF5.h

```diff
@@ -19,9 +19,6 @@ Property::Properties load(const HighFive::Group& group);
 class MorphologyHDF5
 {
   public:
-    MorphologyHDF5(const std::string& uri)
-        : _err(uri)
-        , _uri(uri) {}
```

Comment:
@matz-e I just realized that this commit broke the `uri` which no longer contains the morphology filename. This is espicially useful in case MorphIO fails because it used to tell you which file failed. Do you know what was the issue with this ctor ?

Reply:
IIRC, that `HighFive::Group` has no default constructor…

Actually, recovering the filename should be possible with HighFive now, so maybe this could be reworked.